pkg/cmd/util/output/backup_describer.go

```diff
@@ -467,3 +479,43 @@ func (v *volumesByPod) Sorted() []*podVolumeGroup {
 
 	return v.volumesByPodSlice
 }
+
+func DescribeCSIVolumeSnapshots(d *Describer, details bool, volumeSnapshotContents []snapshotv1beta1api.VolumeSnapshotContent) {
+	if !features.IsEnabled(velerov1api.CSIFeatureFlag) {
+		return
+	}
+
+	if !details {
+		d.Printf("CSI Volume Snapshots (specify --details for more information)\n")
```

Comment:
Maybe we should display a simple count of CSI snapshots here, if `details == false`, similarly to what's being done for non-CSI snapshots.